site/styles/_awards.scss

```diff
@@ -3,9 +3,10 @@
 
   &.meta-awards-all {
     display: grid;
-    gap: 30px 104px;
+    gap: 28px 104px;
+    grid-auto-rows: 1fr;
     margin-bottom: 0;
-    margin-top: 20px;
+    margin-top: 30px;
```

Comment:
I would prefer if this was a margin-bottom on the cta buttons container for consistency.  Delivery Team decided ages ago that core-template should only use margin-bottom where possible because of the conditional rendering to avoid the problem you've got here e.g. a film has awards so the gap between the cta buttons and the awards is 30px... but a film that doesn't have awards will then have a 12px gap between the cta buttons and the synopsis... looks inconsistent.

Reply:
Have reverted this change to get over the line. Will consider margin bottom approach for this in next round of spacing changes.